app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/MessageCompositionInputStateHolder.kt

```diff
@@ -90,9 +93,15 @@ class MessageCompositionInputStateHolder(
         }
     }
 
-    fun handleOffsetChange(offset: Dp, navBarHeight: Dp) {
+    fun handleOffsetChange(offset: Dp, navBarHeight: Dp, source: Dp, target: Dp) {
         val actualOffset = max(offset - navBarHeight, 0.dp)
 
+        // this check secures that if some additional space will be added to keyboard
+        // like gifs search it will save initial keyboard height
```

Comment:
question: what if the configuration changes, like rotating the screen or changing to a different keyboard? Then the initial height of the previous one will still be kept instead of update and adjust to the new configuration - users don't change keyboards very often and right now we don't support rotations, but if we decide to do so, it may be more painful to fix later than now

Reply:
I think it will not because I'm not saving it to saveable 🤔